pkg/controller/controller_binding.go

```diff
@@ -349,38 +352,67 @@ func isPlanBindable(serviceClass *v1alpha1.ServiceClass, plan *v1alpha1.ServiceP
 
 func (c *controller) injectBinding(binding *v1alpha1.Binding, credentials map[string]interface{}) error {
 	glog.V(5).Infof("Creating Secret %v/%v", binding.Namespace, binding.Spec.SecretName)
```

Comment:
Is this comment misleading since it appears that this func is also used to update a Binding's secret? The func name itself is a bit off too - perhaps "upsertBinding" ?